test/index.ts

```diff
@@ -0,0 +1,46 @@
+import reactAxe from '../';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+// default use
+reactAxe(React, ReactDOM, 1000);
```

Comment:
I don't believe the timeout is _required_. We should also have something like:

```ts
reactAxe(React, ReactDOM)
```

Reply:
I thought so too but if you use the option you have to pass it and I didn't feel like updating the code to make that work https://github.com/dequelabs/react-axe/pull/171#issuecomment-647587840

Reply:
Interesting, OK. I guess the timeout is required then. We should probably make it an optional parameter with a sensible default in a future release.